docs/integrations/python.md

```diff
@@ -68,102 +66,293 @@ configuration.proxy = <proxy server URL>
 
 Now that you have a client object, you can use it to interact with the API.
 
+
+To shorten the code example, let's create a helper function that will iterate over all paginated API's results:
+
+```python
+def pagination_helper(page_fetcher, **kwargs):
+    """Helper function to iterate over paginated results"""
+    while True:
+        resp = page_fetcher(**kwargs)
+        yield from resp.results
+        if not resp.pagination.has_more:
+            break
+        kwargs['after'] = resp.pagination.next_offset
+```
+
 #### Creating a repository
 
 ```python
-repo = models.RepositoryCreation(name='example-repo', storage_namespace='s3://storage-bucket/repos/example-repo', default_branch='main')
-client.repositories_api.create_repository(repo)
-# output:
-# {'creation_date': 1617532175,
-#  'default_branch': 'main',
-#  'id': 'example-repo',
-#  'storage_namespace': 's3://storage-bucket/repos/example-repo'}
+import lakefs_sdk
+from lakefs_sdk.models import RepositoryCreation
+from lakefs_sdk.client import LakeFSClient
+
+configuration = lakefs_sdk.Configuration(
+    host="http://localhost:8000",
+    username="AKIAIOSFODNN7EXAMPLE",
+    password="wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
+)
+client = LakeFSClient(configuration)
```

Comment:
Not a fan of including this code each time: it makes it much harder for me to read the important bit, and actually also harder for me to copy-paste into a REPL.  Even if I am writing code, typically I will want to perform more than one operation, and I will still need to edit away most of the lines.